frontend/src/router/index.js

```diff
@@ -509,6 +509,7 @@ export default function createRouter ({ store, userManager }) {
       await Promise.all([
         ensureCloudProfilesLoaded(),
         ensureProjectsLoaded(),
+        store.dispatch('fetchKubeconfigData'),
```

Comment:
Shouldn‘t we call it only for the accounts page?

Reply:
The method `fetchKubeconfigData` only makes a remote call if `state.kubeconfigData` is empty.